### ISSUE TITLE
Command "yarn run browser" fails due to non-available context files

### ISSUE THREAD

**Issue by larsgsvensson** [comunica/comunica#155]
#### Issue type:

- :bug: Bug

____
#### Description:

When I execute `yarn run browser` in `components/actor-init-sparql` the build fails with the message 'Error: Invalid components file "/home/svensson/comunica/packages/actor-init-sparql/components/Actor/Init/Sparql.jsonld"' and later 'jsonld.InvalidUrl: Dereferencing a URL did not result in a valid JSON-LD object.'.
In [`components/Actor/Init/Sparql.jsonld`](https://github.com/comunica/comunica/blob/master/packages/actor-init-sparql/components/Actor/Init/Sparql.jsonld) the context refers to the files `https://linkedsoftwaredependencies.org/contexts/comunica-actor-init-sparql.jsonld` and `https://linkedsoftwaredependencies.org/contexts/comunica-bus-init.jsonld`. Both URLs return 404 when dereferenced.

____
#### Environment:

Ubuntu 16.04

#### Crash log:

https://pastebin.com/pu982xgP

**Comment by rubensworks** [comment]
Hmm, this should work just fine on Ubuntu, which is the OS our CI runs on as well.

Are you sure that you installed the dev environment by running `yarn install` in the root, and not in the `actor-init-sparql` folder?
If so, could you send the the `ls -al` output of your `actor-init-sparql` folder?

The URL dereferencing issue is being tracked here: #149.

**Comment by larsgsvensson** [comment]
Hm, this is weird -- now it doesn't build at all...

Here's what I do:
```bash
svensson@ldslab:~$ rm -rf comunica/
svensson@ldslab:~$ git clone https://github.com/comunica/comunica.git
Cloning into 'comunica'...
remote: Counting objects: 21274, done.
remote: Compressing objects: 100% (191/191), done.
remote: Total 21274 (delta 203), reused 26 (delta 23), pack-reused 21060
Receiving objects: 100% (21274/21274), 7.30 MiB | 6.08 MiB/s, done.
Resolving deltas: 100% (18937/18937), done.
Checking connectivity... done.
svensson@ldslab:~$ cd comunica/
svensson@ldslab:~/comunica$ yarn install
ver -> 8.11.2
yarn install v1.7.0
$ node -e "if (!fs.existsSync('packages/runner-cli/bin/run.js')) { var os = fs.createWriteStream('packages/runner-cli/bin/run.js'); os.write('#!/usr/bin/env node\nconsole.error(\'Temporary runner script during lerna installation. If you see this, please run yarn install again.\');\nprocess.exit(1);'); os.end(); }"
[1/5] Validating package.json...
[2/5] Resolving packages...
â  jest@^23.1.0
warning jest > jest-cli > istanbul-api > istanbul-lib-hook@1.2.1: 1.2.0 should have been a major version bump
â  is-extendable@^1.0.1
[3/5] Fetching packages...
info fsevents@1.1.3: The platform "linux" is incompatible with this module.
info "fsevents@1.1.3" is an optional dependency and failed compatibility check. Excluding it from installation.
[4/5] Linking dependencies...
warning " > babel-loader@7.1.2" has unmet peer dependency "webpack@2 || 3".
[5/5] Building fresh packages...
success Saved lockfile.
$ lerna bootstrap && tsc
lerna info version 2.5.1
lerna info Bootstrapping 86 packages
lerna info lifecycle preinstall
lerna info Installing external dependencies
lerna ERR! execute callback with error
lerna ERR! Error: Command failed: yarn install --mutex network:42424 --non-interactive --no-lockfile
...
error Command failed with exit code 1.
```
Full message at https://pastebin.com/RupBJJv2

(Apologies if I've overseen anything trivial, I'm new to yarn)

**Comment by rubensworks** [comment]
It looks like your node-gyp is failing on rdf-canonize, which is a dependency of JSON-LD.
This is a known issue on some systems: https://github.com/digitalbazaar/jsonld.js/issues/232

Quote from the issue:
> solved by updating GCC from 4.8.2 to 4.9.2

**Comment by larsgsvensson** [comment]
Updateing GCC to 4.9 solved the problem, now `yarn run browser` creates `comunica-browser.js`. Thank for your help!
